editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -960,16 +947,15 @@ export const GridControl = React.memo<GridControlProps>(({ grid }) => {
           const countedColumn = Math.floor(cell % grid.columns) + 1
           const id = gridCellTargetId(grid.elementPath, countedRow, countedColumn)
           const borderID = `${id}-border`
-          const dotgridColor =
-            activelyDraggingOrResizingCell != null ? features.Grid.dotgridColor : 'transparent'
+          const dotgridColor = activelyDraggingOrResizingCell != null ? `#00000033` : 'transparent'
 
           const isActiveCell =
             countedColumn === currentHoveredCell?.column && countedRow === currentHoveredCell?.row
 
           const borderColor =
             isActiveCell && targetsAreCellsWithPositioning
               ? colorTheme.brandNeonPink.value
-              : features.Grid.inactiveGridColor
+              : `#00000033`
```

Comment:
This should ideally be coming from the theme

Reply:
yep, but like the other counterpart (line 950) we currently miss a low-opacity black, will have a separate PR to fix both